packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -7520,6 +7520,7 @@ export interface ITagItemProps extends IPickerItemProps<ITag> {
     enableTagFocusInDisabledPicker?: boolean;
     styles?: IStyleFunctionOrObject<ITagItemStyleProps, ITagItemStyles>;
     theme?: ITheme;
+    title?: string;

```

Comment:
An example using this new title prop would be helpful

Reply:
@aneeshack4 I added an example as part of the tests. Does that suit you?

Reply:
Yes since there aren't any existing examples on the TagPicker page on the docs site this is fine.